src/components/select/select.component.ts

```diff
@@ -324,8 +324,8 @@ export class SelectComponent<T> implements OnInit, OnChanges, OnDestroy, Control
             event.preventDefault();
         }
 
-        // when the user types and the value is not empty then we should open the dropdown
-        if (event.keyCode !== ESCAPE) {
+        // when the user types and the value is not empty then we should open the dropdown except for the tab and escape keys.
+        if (!(event.keyCode === ESCAPE || event.keyCode === TAB)) {

```

Comment:
That has partially fixed the issue, however if I tab to the select an type 'U' the list appears. If I tab out and tab back in and type 'U' the list does not reappear, eg:

[![Image from Gyazo](https://i.gyazo.com/bdb73b034049047bec3671952700698d.gif)](https://gyazo.com/bdb73b034049047bec3671952700698d)